app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb

```diff
@@ -9,8 +9,8 @@
 
         <%= render Content::ChecklistCollageComponent.new(
           checklist: [
-            "You'll earn at least £30k (or more in London) if you're just starting as a teacher.",
-            "The average salary for a classroom teacher in England is over £40k a year."
+            "You'll earn at least #{v :salaries_starting_minshortened } (or more in London) if you're just starting as a teacher.",
+            "The average salary for a classroom teacher in England is over #{ v :salaries_average_classroomshortened } a year."
```

Comment:
```suggestion
            "The average salary for a classroom teacher in England is over  <%= v :salaries_average_classroomshortened %> a year."
```

Reply:
Resolving this conversation - this isn't the right way to fix the linting errors (sorry) as they are already within a `<%=` in line 10